dart/math/Geometry.h

```diff
@@ -533,6 +533,35 @@ Eigen::Vector2d computeClosestPointOnSupportPolygon(
     const Eigen::Vector2d& _p,
     const SupportPolygon& _support);
 
+
+// Represents a bounding box with minimum and maximum coordinates.
+class BoundingBox {
```

Comment:
I think that's very confusing. All bounding boxes (including OBBs) are axis aligned in their own reference frame. In practice these are actually OBBs since they are attached to a moving reference frame (e.g a body node)